QUEST.tsv

```diff
@@ -44,9 +44,9 @@ QUEST_20150317_000043	Force start?
 QUEST_20150317_000044	Items to be deleted
 QUEST_20150317_000045	Select GenType
 QUEST_20150317_000046	Select Coordinates
-QUEST_20150317_000047	Knight Titas
-QUEST_20150317_000048	If that is so then go see Knight Commander Uska at Klaipeda. {nl}But first, I must speak with you. Follow me.
-QUEST_20150317_000049	It would be nice if I had a moment to rest…
+QUEST_20150317_000047	$Knight Titas
+QUEST_20150317_000048	$If so, go and see Knight Commander Uska, at Klaipeda. {nl}But first, I have an earnest request that I'd like to you to attend.
```

Comment:
If so, go and see Knight Commander Uska, at Klaipeda -> If so, go and see Knight Commander Uska in Klaipeda.

Replace the 'at' preposition with 'in'.